### ISSUE TITLE
[Bug] Server goes down and can not recover when parsing the library

### ISSUE THREAD

**Issue by jwillmer** [gotson/komga#370]
### Komga environment
- **OS**: 
- **Komga version**: 
- [x] **I am running Komga with Docker**
  - Docker image tag `latest`

### Describe the bug
Server crash when parsing the library

### Steps to reproduce
- Follow the docker-compose installation guide.
- Login and select the book folder as library
- While processing the files the server throws an exception

### Behavior
Server goes down and can not recover (boot loop)

### Screenshots
![image](https://user-images.githubusercontent.com/1503577/103418616-c4da9d00-4b8f-11eb-9b12-7c0eb16184d5.png)

### Log file
https://gist.github.com/jwillmer/0155218074b6d02b35cca98988a33853

**Comment by gotson** [comment]
Please provide a complete log file.

**Comment by jwillmer** [comment]
@gotson It is the complete log. Before was a long list of books that were added and after was the exception repeating since I enabled automatic restarts.

**Comment by gotson** [comment]
It's not a complete log, I need one from startup.

**Comment by jwillmer** [comment]
After killing the docker instance I recreated it and waited till the logs did not show any book processing anymore. After that I logged in and all worked. I guess the app had too much to do and failed. With my library now imported I can't reproduce the error. Will close this issue for now.

Off Topic: How do I sort my comics so that the app understands series. I used calibre to manage the comics but in the app each book is it's own series.
